package io.usoamic.validateutilkt.exception

class InvalidValueError : ValidateUtilError()